src/dlgIRC.cpp

```diff
@@ -823,6 +834,11 @@ QPair<bool, QString> dlgIRC::writeIrcHostPort(Host* pH, int port)
     return pH->writeProfileData(dlgIRC::HostPortCfgItem, QString::number(port));
 }
 
+QPair<bool, QString> dlgIRC::writeIrcHostSecure(Host* pH, bool secure)
+{
+    return pH->writeProfileData(dlgIRC::HostSecureCfgItem, (secure ? QLatin1String("true") : QLatin1String("false")));
```

Comment:
This works great as of now.  But when it saves `false` to file, it apparently throws a newline in front.  Still functional but confusing to me.  Maybe writeProfileData does something I'm not expecting.
```Alpha@Alpha-F5BZ322 MSYS /c/Users/Alpha/.config/mudlet/profiles/zzz-three
$ cat irc_secure
true
Alpha@Alpha-F5BZ322 MSYS /c/Users/Alpha/.config/mudlet/profiles/zzz-three
$ cat irc_secure

false
```

Reply:
@SlySven ?